.github/workflows/ci.yml

```diff
@@ -50,14 +51,67 @@ jobs:
       - name: Run Cargo.toml linter
         run: git ls-files | grep Cargo.toml$ | xargs --verbose -n 1 cargo-toml-lint
 
-  build:
+  escrow:
```

Comment:
The above comment will also obviate the need for this, which is unmaintainable.

Reply:
I agree that this is spammy but I'm unsure of how to use distinct versions in the matrix. Perhaps a script that outputs the version and pipes it to the required command... not ideal